fv3core/utils/gt4py_utils.py

```diff
@@ -349,6 +380,27 @@ def make_storage_dict(
     return data_dict
 
 
+# def k_slice_operation(key, value, ki, dictionary):
```

Comment:
will we need this later?

Reply:
Happy to remove the comment, is that what you mean?